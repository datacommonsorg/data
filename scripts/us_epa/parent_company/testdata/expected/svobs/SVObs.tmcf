Node: E:EpaParentCompanyStatVarObs->E0
typeOf: dcs:StatVarObservation
observationAbout: C:EpaParentCompanyStatVarObs->parent_company_dcid
variableMeasured: C:EpaParentCompanyStatVarObs->sv_measured
observationPeriod: C:EpaParentCompanyStatVarObs->obs_period
unit: dcs:MetricTonCO2e
measurementMethod: dcs:dcAggregate/EPA_GHGRP
value: C:EpaParentCompanyStatVarObs->value
observationDate: C:EpaParentCompanyStatVarObs->year
