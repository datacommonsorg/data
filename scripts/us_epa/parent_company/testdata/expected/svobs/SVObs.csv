parent_company_dcid,sv_measured,obs_period,value,year
EpaParentCompany/abc,Annual_Emissions_Methane_NonBiogenic,P1Y,997.0,2016
EpaParentCompany/abc,Annual_Emissions_Methane_NonBiogenic,P1Y,888.825,2017
EpaParentCompany/efg,Annual_Emissions_Methane_NonBiogenic,P1Y,101.175,2017
EpaParentCompany/abc,Annual_Emissions_GreenhouseGas_NitricAcidProduction_NonBiogenic,P1Y,100.0,2016
EpaParentCompany/abc,Annual_Emissions_GreenhouseGas_NitricAcidProduction_NonBiogenic,P1Y,127.5,2017
EpaParentCompany/efg,Annual_Emissions_GreenhouseGas_NitricAcidProduction_NonBiogenic,P1Y,22.5,2017
EpaParentCompany/efg,Annual_Emissions_Methane_NonBiogenic,P1Y,13.0,2016
EpaParentCompany/abc,Annual_Emissions_Methane_NonBiogenic,P1Y,7.125,2018
EpaParentCompany/efg,Annual_Emissions_Methane_NonBiogenic,P1Y,7.125,2018
EpaParentCompany/efg,Annual_Emissions_GreenhouseGas_NitricAcidProduction_NonBiogenic,P1Y,50.0,2016
EpaParentCompany/abc,Annual_Emissions_GreenhouseGas_NitricAcidProduction_NonBiogenic,P1Y,25.0,2018
EpaParentCompany/efg,Annual_Emissions_GreenhouseGas_NitricAcidProduction_NonBiogenic,P1Y,25.0,2018
